src/SFA.DAS.AssessorService.Web/Controllers/CertificateBaseController.cs

```diff
@@ -51,6 +51,7 @@ protected async Task<IActionResult> LoadViewModel<T>(string view) where T : Cert
             Logger.LogInformation($"Got Certificate for {typeof(T).Name} requested by {username} with Id {certificate.Id}");
 
             viewModel.FromCertificate(certificate);
+            viewModel.SetStandardHasVersionsAndOptions(certSession);
```

Comment:
This is getting there but I don't think the right idea is to add properties to the base view model and the base controller when it's only used in the check controller.

How about making the LoadViewModel method virtual in the base and overriding in the check controller to call base + do the additional work?